reference/customers.v2.yml

```
@@ -1254,24 +1264,32 @@ components:
                   - price
               amount:
                 type: string
-                description: A float that specifies the value applied to the price modified
-                example: '"5.0000" (Float, Float as String, Integer)'
+                description: A float that specifies the value applied to the price modified.
+                example: 5.0000" (Float, Float as String, Integer)
               price_list_id:
                 type: integer
-                description: 'If a customer group is assigned to a price list,`method` and `amount` are not shown. `type` and `price_list_id` are returned.'
+                description: If a customer group is assigned to a price list,`method` and `amount` are not shown. `type` and `price_list_id` are returned.
                 example: 3
+        date_created:
+          type: string
+          description: Date on which the customer created the group.
```

```suggestion
          description: Date on which the customer group was created.
```
without knowing much about customers -- pretty sure it's the merchant or API that creates a customer group. shoppers don't create them.